src/Extensions/SitewideContentSubsites.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace SilverStripe\SiteWideContentReport\Model;
+
+use SilverStripe\Core\Extension;
+use SilverStripe\SiteWideContentReport\Form\GridFieldBasicContentReport;
+use SilverStripe\SiteWideContentReport\SitewideContentReport;
+use SilverStripe\Subsites\Model\Subsite;
+use SilverStripe\Forms\HeaderField;
+use SilverStripe\Forms\DropdownField;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Versioned\Versioned;
+use SilverStripe\Assets\File;
+use SilverStripe\CMS\Model\SiteTree;
+
+/**
+ * Provides subsite integration for sitewide content report.
+ *
+ * @extends Extension<SitewideContentReport|GridFieldBasicContentReport>
+ */
+class SitewideContentSubsites extends Extension
```

Comment:
class copied from https://github.com/silverstripe/silverstripe-sitewidecontent-report/blob/4.3/src/Model/SitewideContentSubsites.php#L15

Reply:
Note that class isn't deprecated, as the module is just not getting a CMS 6 release. There for there is not need for a CMS 5 PR to say that "this class will be replaced with this other class in subsites: